src/assets/languages/english.json

```diff
@@ -140,6 +140,13 @@
         },
         "messages": {
           "logoutButton": "Logout"
+        },
+        "alerts": {
+          "caching": {
+            "title": "New version available!",
+            "confirmButtonText": "Update",
+            "cancelButtonText": "Cancel"
```

Comment:
What happens if Cancel is clicked? The user never gets the update?

Reply:
@Raxerz when he clicks **Cancel**, he'll not get the update in that session. But in future whenever he'll open the application again, he'll get the update.